packages/cli/src/util/client.ts

```diff
@@ -4,8 +4,13 @@ import { FileSystemStorage } from '../storage';
 export async function createMedplumClient(options: MedplumClientOptions, profileName?: string): Promise<MedplumClient> {
```

Comment:
Sorry, I wish I would have caught this earlier -- In all of the calls to `createMedplumClient`, it looks like the `profileName` parameter always comes from `options`, so we should be able to just use `options.profile` here, right?  And remove the `profileName` parameter entirely?